### ISSUE TITLE
Update Galician translations

### ISSUE THREAD

**Issue by mmontes11** [squidfunk/mkdocs-material#7330]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "gl",
  "action.edit": "Editar esta páxina",
  "action.skip": "Ir ao contido",
  "action.view": "Ver código fonte desta páxina ⬅️",
  "announce.dismiss": "Non volver a mostrar ⬅️",
  "blog.archive": "Arquivo ⬅️",
  "blog.categories": "Categorías ⬅️",
  "blog.categories.in": "en ⬅️",
  "blog.continue": "Continuar lendo ⬅️",
  "blog.draft": "Borrador ⬅️",
  "blog.index": "Voltar ó índice ⬅️",
  "blog.meta": "Metadatos ⬅️",
  "blog.references": "Enlaces relacionados ⬅️",
  "clipboard.copy": "Copiar no cortapapeis",
  "clipboard.copied": "Copiado no cortapapeis",
  "consent.accept": "Aceptar ⬅️",
  "consent.manage": "Xestionar configuración ⬅️",
  "consent.reject": "Rexeitar ⬅️",
  "footer": "Pé",
  "footer.next": "Seguinte",
  "footer.previous": "Anterior",
  "header": "Cabeceira",
  "meta.comments": "Comentarios",
  "meta.source": "Fonte",
  "nav": "Navegación",
  "readtime.one": "1 minuto de lectura ⬅️",
  "readtime.other": "# minutos de lectura ⬅️",
  "rss.created": "RSS feed ⬅️",
  "rss.updated": "RSS feed do contido actualizado ⬅️",
  "search": "Procurar ⬅️",
  "search.placeholder": "Procura",
  "search.share": "Compartir ⬅️",
  "search.reset": "Limpar",
  "search.result.initializer": "Inicializando procura",
  "search.result.placeholder": "Insira un termo",
  "search.result.none": "Sen resultados",
  "search.result.one": "1 resultado atopado",
  "search.result.other": "# resultados atopados",
  "search.result.more.one": "1 máis nesta páxina",
  "search.result.more.other": "# máis nesta páxina",
  "search.result.term.missing": "Falta",
  "select.language": "Seleccionar idioma",
  "select.version": "Seleccionar version",
  "source": "Ir ao repositorio",
  "source.file.contributors": "Contribuidores ⬅️",
  "source.file.date.created": "Creada",
  "source.file.date.updated": "Última actualización",
  "tabs": "Pestanas",
  "toc": "Táboa de contidos",
  "top": "Volver ao principio"
}[key] }}{% endmacro %}
```

### Country flag

:flag_es:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by mmontes11** [comment]
@squidfunk Thanks a lot for this great project! Here some  translations for the Galician language

**Comment by squidfunk** [comment]
Awesome, thanks for providing them! Could you also translate the following?

```
  "readtime.one": "1 min read ⬅️",
  "readtime.other": "# min read ⬅️",
```

The translations would be complete then ☺️

**Comment by mmontes11** [comment]
Sorry, missed these 2. They have been updated in the issue body!

**Comment by squidfunk** [comment]
Thanks again! Updated in c7c8fcb3a.

**Comment by squidfunk** [comment]
Released as part of [9.5.29](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.29).
